packages/core-modules/contracts/modules/OwnerModule.sol

```diff
@@ -2,8 +2,21 @@
 pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/ownership/Ownable.sol";
+import "@synthetixio/core-contracts/contracts/initializable/InitializableMixin.sol";
+import "../interfaces/IOwnerModule.sol";
 
 // solhint-disable-next-line no-empty-blocks
-contract OwnerModule is Ownable {
+contract OwnerModule is Ownable, IOwnerModule, InitializableMixin {
+    function _isInitialized() internal view override returns (bool) {
+        return _ownableStore().owner != address(0);
+    }
 
+    function isOwnerModuleInitialized() external view override returns (bool) {
+        return _isInitialized();
+    }
+
+    function initializeOwnerModule(address initialOwner) external override onlyIfNotInitialized {
+        nominateNewOwner(initialOwner);
+        acceptOwnership();
+    }
```

Comment:
Wouldn't this have the same functionality, having in mind that the call to `acceptOwnership();` uses `msg.sender`?:
```suggestion
    function initializeOwnerModule() external override onlyIfNotInitialized {
        nominateNewOwner(msg.sender);
        acceptOwnership();
    }
```

Reply:
As discussed in Discord. 

We prefer to have the address explicitly. One caveat is that using the initialize we won't be able to use an EOA to nominate a multisig or gov entity since it will use the msg.sender to accept the ownership in line 20.
But having an _acceptOwnershipBy(address) can allow us to use a different address in the initializer and automatically accept the ownership by that address, but is risky since a typo there can set the owner to an invalid address.

Reply:
As am afterthought.

If we want to set the owner to a Mutlisig or Gov Module, we should first initialize the OwnerModule to the EAO running the deploy and then set the new owner via usual way: Calling nominateNewOwner(multisigAddress) and as a post-deploy accept the ownership by the multisig or govModule using acceptOwnership().

The good part is that we ensure the OwnerModule is initialized to a known address.

Reply:
I like that option! Is less prone to errors during the initialization